docs/source/usage/workflows/quasiNeutrality.rst

```diff
@@ -0,0 +1,70 @@
+.. _usage-workflows-quasiNeutrality:
+
+Quasi-Neutral Initialization
+----------------------------
+
+.. sectionauthor:: Axel Huebl
+
+In order to initialize the electro-magnetic fields self-consistently, one needs to fulfill Gauss's law :math:`\vec \nabla \cdot \vec E = \frac{\rho}{\epsilon_0}` (and :math:`\vec \nabla \cdot \vec B = 0`).
+The trivial solution to this equation is to start *field neutral* by microscopically placing a charge-compensating amount of free electrons on the same position as according ions.
+
+Fully Ionized Ions
+""""""""""""""""""
+
+For fully ionized ions, just use ``ManipulateDeriveSpecies`` in :ref:`speciesInitialization.param <usage-params-core>` and derive macro-electrons :math:`1:1` from macro-ions but increase their weighting by :math:`1:Z` of the ion.
+
+.. code-block:: cpp
+
+   using InitPipeline = mpl::vector<
+       // density profile from density.param and
+       //     start position from particle.param
+       CreateDensity<
+           densityProfiles::YourSelectedProfile,
+           startPosition::YourStartPosition,
+           Carbon
+       >,
+       // create a macro electron for each macro carbon but increase its
+       //     weighting by the ion's proton number so it represents all its
+       //     electrons after an instantanous ionization
+       ManipulateDeriveSpecies<
+           manipulators::ProtonTimesWeighting,
+           Carbon,
+           Electrons
+       >
+   >;
+
+Partly Ionized Ions
+"""""""""""""""""""
+
+For partial pre-ionization, the :ref:`FoilLCT example <usage-examples-foilLCT>` shows a detailed setup.
+First, define a functor that manipulates the number of bound electrons in :ref:`particle.param <usage-params-core>`, e.g. to *once ionized*.
+Then again in :ref:`speciesInitialization.param <usage-params-core>` set your initialization routines to:
+
+.. code-block:: cpp
+
+   using InitPipeline = mpl::vector<
+       // density profile from density.param and
+       //     start position from particle.param
+       CreateDensity<
+           densityProfiles::YourSelectedProfile,
+           startPosition::YourStartPosition,
+           Carbon
+       >,
+       // partially pre-ionize the carbons by manipulating the carbon's
+       //     `boundElectrons` attribute,
+       //     functor defined in particle.param: set to C^1+
```

Comment:
it would look more consistent if you'd either omit the `^` or put `1+` in curly braces:   
`C1+` or `C^{1+}`